packages/auth/src/tests/app.ts

```diff
@@ -0,0 +1,37 @@
+import Knex from 'knex'
+import nock from 'nock'
+import { IocContract } from '@adonisjs/fold'
+
+import { start, gracefulShutdown } from '..'
+import { App, AppServices } from '../app'
+
+export interface TestContainer {
+  port: number
+  app: App
+  knex: Knex
+  connectionUrl: string
+  shutdown: () => Promise<void>
+}
+
+export const createTestApp = async (
+  container: IocContract<AppServices>
+): Promise<TestContainer> => {
+  const config = await container.use('config')
+  config.port = 0
+
```

Comment:
You could suppress test logging with:
https://github.com/interledger/rafiki/blob/main/packages/backend/src/tests/app.ts#L40-L49

Reply:
bump

Reply:
I think the log suppression is just below this comment?

Reply:
Oops, I'm apparently overly dependent on GitHub marking review comments as `Outdated`